### PR TITLE
Fixing a compiler warning.

### PR DESCRIPTION
**Pull request by s-maheshbabu**
Fixing a compiler warning.

### CODE REVIEW

File: src/SpeakDirectiveParser.js [hunk 1]
```diff
@@ -31,11 +31,11 @@ export function extractAlexaTextResponse(alexaRawResponse) {
     !parsedResponse ||
     !parsedResponse.multipart ||
     !parsedResponse.multipart[0].body ||
-    parsedResponse.multipart[0].body.length == 0
+    parsedResponse.multipart[0].body.length === 0
   ) {
     throw new IllegalArgumentError(
       "Given response is not a valid multi-part message. Input: " +
-      alexaRawResponse
+        alexaRawResponse
```
Comment (gootla-manasa) [inline src/SpeakDirectiveParser.js]: why are these formatting changes here?

File: src/SpeakDirectiveParser.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ export function extractAlexaTextResponse(alexaRawResponse) {
   if (!hasIn(avsDirective, ["directive", "payload", "caption"]))
     throw new IllegalArgumentError(
       "Given directive doesn't contain the expected path directive.payload.caption. Input: " +
-      avsDirective
+        avsDirective
```
Comment (gootla-manasa) [inline src/SpeakDirectiveParser.js]: Please refrain from making any formatting related changes.

File: src/SpeakDirectiveParser.js (hunk 1 above)
Comment (s-maheshbabu) [reply src/SpeakDirectiveParser.js]: Done.

File: src/SpeakDirectiveParser.js (hunk 2 above)
Comment (s-maheshbabu) [reply src/SpeakDirectiveParser.js]: Done.

**Pull request merged**